tests/selenium_tests/test_new_excerpt.py

```diff
@@ -26,10 +26,13 @@ def test_new_excerpt(base_url, login, file_name, file_format, selenium, reload_u
     create.send_keys(Keys.RETURN)
 
     # wait until download link appears
-    selenium.find_element_by_link_text('↻ Reload')
-    element = reload_until_condition(selenium.find_element_by_class_name, "form-control")
+    selenium.find_element_by_xpath("/html/body/div/div/div[2]/div[2]/div[1]/div/div[1]/h3")
+    first_a = "/html/body/div/div/div[2]/div[2]/div[1]/div/div[2]/div[1]/div[1]/div[2]/div/div[1]/p/a"
```

Comment:
The reason was simply that we need to get the first of the div's and select it in there. I haven't tried the same with `find_element_by_link_text` because the tests take such a very long time to run, but will take a look if it might be possible to do so.